common/src/main/java/io/novafoundation/nova/common/data/FileProviderImpl.kt

```diff
@@ -13,32 +11,26 @@ class FileProviderImpl(
     private val context: Context
 ) : FileProvider {
 
-    override suspend fun getFileInExternalCacheStorage(fileName: String): File {
-        return withContext(Dispatchers.IO) {
-            val cacheDir = context.externalCacheDir?.absolutePath ?: directoryNotAvailable()
+    override fun getFileInExternalCacheStorage(fileName: String): File {
```

Comment:
Removed suspend modifier since it's useless and limits us from using it outside viewmodel